projects/storyteller/webapp/src/app/components/feed/feed.component.scss

```diff
@@ -1,70 +1,4 @@
+@import 'app/styles/feeds.scss';
```

Comment:
Wow, it's absolute way to import styles! Cool!

Reply:
Thanks vadim!! I wanted to ask  if I do it right...😀😀